test/apps/react-oie/vite.config.js

```diff
@@ -11,7 +11,7 @@ const env = {};
 // List of environment variables made available to the app
 ['ISSUER', 'SPA_CLIENT_ID'].forEach((key) => {
   if (!process.env[key]) {
-    throw new Error(`Environment variable ${key} must be set. See README.md`);
+    console.warn(`Environment variable ${key} should be set for development. See README.md`);
```

Comment:
can the app actually run without these env vars?

Reply:
During e2e, the app can start without oidc config, then the app will be updated with query params with the newly generated test app.